libs/openFrameworks/utils/ofConstants.h

```diff
@@ -389,6 +389,12 @@ typedef TESSindex ofIndexType;
 	#endif
 #endif
 
+#if (defined(_M_ARM64) || defined(_M_ARM64EC)) && defined(TARGET_WIN32)
+	#undef USE_FMOD // No FMOD lib for ARM64 yet
+	#define TARGET_NO_SOUND
```

Comment:
Fmod not supported on ARM64/ARM64EC currently

Reply:
We have a Windows Media Foundation sound player now. 
So you could skip `#define TARGET_NO_SOUND` and it should work.

Reply:
Oh yes! I just tested on ARM64 and yes that works with change to  #define OF_NO_FMOD which activates now into that media foundation! Awesome!